cmd/util/cmd/common/clusters.go

```diff
@@ -174,13 +176,44 @@ func ConvertClusterQcsCdc(qcs []*flow.QuorumCertificate, clusterList flow.Cluste
 		if err != nil {
 			return nil, fmt.Errorf("could not decode signer indices: %w", err)
 		}
-		voteData[i] = &flow.ClusterQCVoteData{
-			SigData:  qc.SigData,
-			VoterIDs: voterIds,
+		cdcVoterIds := make([]cadence.Value, len(voterIds))
+		for i, id := range voterIds {
+			cdcVoterIds[i] = cadence.String(id.String())
 		}
+
+		qcVoteData[i] = cadence.NewStruct([]cadence.Value{
+			// voteSignatures
+			cadence.String(fmt.Sprintf("%#x", qc.SigData)),
+			// voterIDs
+			cadence.NewArray(cdcVoterIds).WithType(cadence.NewVariableSizedArrayType(cadence.StringType)),
+		}).WithType(voteDataType)
+
 	}
 
-	return voteData, nil
+	return qcVoteData, nil
+}
+
+// newFlowClusterQCVoteDataStructType returns the FlowClusterQC cadence struct type.
+func newFlowClusterQCVoteDataStructType(clusterQcAddress string) *cadence.StructType {
+
+	// FlowClusterQC.ClusterQCVoteData
+	address, _ := cdcCommon.HexToAddress(clusterQcAddress)
+	location := cdcCommon.NewAddressLocation(nil, address, "FlowClusterQC")
+
+	return &cadence.StructType{
+		Location:            location,
+		QualifiedIdentifier: "FlowClusterQC.ClusterQCVoteData",
+		Fields: []cadence.Field{
+			{
+				Identifier: "voteSignatures",
```

Comment:
I suspect we need to change this in Cadence as well but naming `voteSignatures` makes it confusing for me since it's an aggregated signature. A single signature essentially.

Reply:
https://github.com/onflow/flow-go/pull/6156/commits/9e062d18786e92a73c89da141cb6e3814bb8effa